package.json

```diff
@@ -33,7 +33,9 @@
     "predocs": "npm run compile",
     "docs": "jsdoc -c .jsdoc.js",
     "generate-scaffolding": "repo-tools generate all && repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json",
-    "system-test": "mocha build/system-test --timeout 600000",
+    "system-test": "mocha build/system-test --timeout 600000 && npm run conformance-test",
+    "conformance-test": "mocha build/conformance-test --timeout 600000",
+    "preconformance-test": "npm run compile",
```

Comment:
It's cool that this is here, but what's invoking it?  How does this relate to the current CI?

Reply:
Right now I'm just running it after system-test (see `system-test` run-script)